go/vt/vtctl/vtctl.go

```diff
@@ -798,6 +801,26 @@ func commandDeleteTablet(ctx context.Context, wr *wrangler.Wrangler, subFlags *f
 	return nil
 }
 
+func commandDemoteMasterTablet(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
+	if err := subFlags.Parse(args); err != nil {
+		return err
+	}
+	if subFlags.NArg() == 0 {
+		return fmt.Errorf("the <tablet alias> argument must be used to specify at least one tablet when calling the DemoteMasterTablet command")
+	}
+
+	tabletAliases, err := tabletParamsToTabletAliases(subFlags.Args())
+	if err != nil {
+		return err
+	}
+	for _, tabletAlias := range tabletAliases {
+		if err := wr.DemoteMasterTablet(ctx, tabletAlias); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
```

Comment:
I assume this new command is only used for testing? But this still exposes the command to everyone and can confuse people.

Reply:
Yes, we had a discussion regarding this and have decided to remove this function. We will create a separate function which demotes a primary, which isn't exposed for end users.

Reply:
Moved this functionality to a local function in the testing environment